recipe/run_test.sh

```diff
@@ -3,20 +3,31 @@
 pushd $RECIPE_DIR/test_data/
 
 # From @mhearne-usgs. See https://github.com/conda-forge/gdal-feedstock/issues/23#issue-144997326
-proj4="+y_0=2400761.714982585 +lat_ts=-19.6097 +a=6378137.0 +proj=merc +units=m +b=6356752.3142 +lat_0=-19.6097 +x_0=-0.0 +lon_0=-70.7691"
-
-gdalwarp -s_srs "+proj=latlong" -t_srs "$proj4" -of EHdr grid.asc grid.flt
-
-# Test ISIS3/USGS driver `SetNoDataValue()` issue.
+echo ""
+echo "GDAL WARP TEST"
+echo ""
+# proj4="+y_0=2400761.714982585 +lat_ts=-19.6097 +a=6378137.0 +proj=merc +units=m +b=6356752.3142 +lat_0=-19.6097 +x_0=-0.0 +lon_0=-70.7691"
+# gdalwarp -s_srs "+proj=latlong" -t_srs "$proj4" -of EHdr grid.asc grid.flt
```

Comment:
This test is failing but my guess us some API change. I'll take a closer look later today.

Reply:
@mhearne-usgs this test now returns `ERROR 1: Couldn't determine X spacing` do you know if `gdal 2.3.0` changed the API here or is this a real failure?